tests/integration/utils.js

```diff
@@ -175,6 +175,26 @@ testUtils.isCouchDB = function (cb) {
   });
 };
 
+testUtils.getServerType = function (cb) {
+  const knownServers = [
+    'couchdb',
+    'express-pouchdb', // seems to be pouchdb-server
```

Comment:
```suggestion
    'express-pouchdb',
```
It lives in pouchdb-server`s packages, but it just wraps what you feed it with.
https://github.com/pouchdb/pouchdb-server/blob/975ea2377957223dc828bcdb4f5d403a5cc9f709/packages/node_modules/express-pouchdb/lib/index.js#L251